example/src/main/scala/xs4s/example/brief/BriefZIOExample.scala

```diff
@@ -0,0 +1,39 @@
+package xs4s.example.brief
+
+import java.io.IOException
+import zio.blocking.Blocking
+
+/** An example that compiles but does not run - for use in the README */
+object BriefZIOExample {
+  import javax.xml.stream.events.XMLEvent
+  import xs4s._
+  import xs4s.ziocompat._
+  import xs4s.syntax.zio._
+  import zio._
+  import zio.stream._
+
+  import scala.xml.Elem
+
+  /**
+    *
+    * @param byteStream Could be, for example, zio.stream.Stream.fromInputStream(inputStream)
+    * @return
+    */
+  def extractAnchorTexts[R <: Blocking](byteStream: ZStream[R, IOException, Byte]): ZStream[R, Throwable, String] = {
+    /** extract all elements called 'anchor' **/
+    val anchorElementExtractor: XmlElementExtractor[Elem] =
+      XmlElementExtractor.filterElementsByName("anchor")
+
+    /** Turn into XMLEvent */
+    val xmlEventStream: ZStream[R, Throwable, XMLEvent] =
+      byteStream.via(byteStreamToXmlEventStream()(_))
```

Comment:
should I add the "meaningful" arg here too?

Reply:
Either that, or actually do you need to do the `(_)` at all? I think it should compile without.

Reply:
I would have thought so, but it wasn't working, I want to figure out why to cleanup that syntax